api/server/structs/conversions.go

```diff
@@ -52,6 +52,9 @@ func HistoricalSummaryFromConsensus(s *eth.HistoricalSummary) *HistoricalSummary
 }
 
 func (s *SignedBLSToExecutionChange) ToConsensus() (*eth.SignedBLSToExecutionChange, error) {
+	if s.Message == nil {
+		return nil, errNilValue
```

Comment:
```suggestion
		return nil, server.NewDecodeError(errNilValue, "Message")
```
Can you do the same in other places where you only return `nil, errNilValue`?